docs/developer/index.rst

```diff
@@ -12,8 +12,7 @@ Developer
     using_omnisci
     using_pyarrow_on_ray
     using_sql_on_ray
-    troubleshooting
 
 .. meta::
     :description lang=en:
-        Developer-specific documentation.
\ No newline at end of file
+        Developer-specific documentation.
```

Comment:
What's the change here? The line shows up as a diff on GitHub, but it looks the same to me.

Reply:
There isn't one, but for some reason the last line gets highlighted in two of the .rst files